### PR TITLE
Parametrise session properties

### LINKED ISSUE #607: Session creation is slow
Hi,

When running small job through lighter, I observed that the scheduling of the job by lighter takes time.
I use only Session.

If I run spark-pi with args=1 the single spark stage takes 0.1ms.
If I run it directly trough a spark-submit batch on my k8s cluster it takes 23s to complete.

However, If I have to create a session with lighter, it takes ~90s before the session is in state="STARTED" or IDLE.

Is it due to micronauts scheduling things every minutes ?
Something else ?
Is it tunable ?

In addition, If I 
"Create a session and then wait for status = STARTED/IDLE and then submit a statement" ~ 90s

it is way slower than:
"Create a session and submit a statement on the new session not started yet and wait for statement state to be available" ~ 75s

Thanks and regards

### PR DESCRIPTION
**Pull request by Minutis**
Parametrise interval for scheduling of new sessions
Parametrise interval for tracking running sessions

Closes #607

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -12,8 +12,10 @@ Lighter can be configured by using environment variables. Currently, Lighter sup
...
 | LIGHTER_PY_GATEWAY_PORT                | Port for live Spark session communication                                                                                                        | 25333                           |
 | LIGHTER_URL                            | URL which can be used to access Lighter form Spark Job                                                                                           | http://lighter.spark:8080       |
-| LIGHTER_SESSION_TIMEOUT_MINUTES        | Session lifetime in minutes (from last statement creation). Use negative value to disable                                                        | 90                              |
-| LIGHTER_SESSION_TIMEOUT_ACTIVE         | Should Lighter kill sessions with waiting statements (obsolete when `LIGHTER_SESSION_TIMEOUT_MINUTES` is negative)                               | false                           |
+| LIGHTER_SESSION_TIMEOUT_INTERVAL       | `java.time.Duration` representing session lifetime (from last statement creation). Use `0` value to disable                                      | 90m                             |
```
Comment (pdambrauskas) [inline docs/configuration.md:15]: ‘0’ as a default value would probably wont work for Java Duration?

Comment (Minutis) [reply docs/configuration.md]: I did change the documentation to use `0m`. Although what's strange, that it should work with `0` also, but at least in groovy tests I got `NULL` value instead of Duration.ZERO which I expected and which I get with `0m`. I might be wrong but [documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.conversion.durations) states that using plain number should default to milliseconds.
